### DISCUSSION TITLE
请问该项目可以输出坐标吗

### DISCUSSION THREAD

**Discussion by KrisTopherHao** [RapidAI/TableStructureRec#10] [Q&A]
你好，我尝试debug查看坐标数据，然后定位到bbox_decode方法和postprocess方法，发现不管什么图片坐标数组都是10000条数据？请问我想得到表格框的坐标以及框内文本坐标在哪里可以获得，非常感谢！

**Comment by SWHL** [comment]
目前该仓库两个算法还没支持，后续会更新。
[rapid_table](https://github.com/RapidAI/RapidStructure/blob/main/docs/README_Table.md)是支持的，建议先用那个

**Reply by KrisTopherHao** [reply]
有线wired_table_rec，模型cycle_center_net_v1.onnx，就是用demo_wired.py测试的

**Reply by KrisTopherHao** [reply]
好的，感谢

**Reply by phamkhactu** [reply]
> 有线wired_table_rec，模型cycle_center_net_v1.onnx，就是用demo_wired.py测试的

Hi all, 

How can I get cycle_center_net_v1.onnx, I saw the [pretrained models](https://github.com/AlibabaResearch/AdvancedLiterateMachinery/tree/main/DocumentUnderstanding/LORE-TSR#21-pretrained-models) and converted ckpt_wireless to onnx as @SWHL suggested. The onnx model is a ```LinelessTableRecognition```.  Can I use ```ConvertLOREToONNX```  to convert ```cv_dla34_table-structure-recognition_cycle-centernet``` as ```cycle_center_net_v1```  onnx model?   I am very happy for your sugesstion.

**Reply by Joker1212** [reply]
@KrisTopherHao 我们更新了模型，目前两个模型都会输出相同的数据，包含html,耗时，逻辑坐标，物理坐标，ocr识别的物理坐标，并提供两个方法可视化排序后的ocr坐标结果和表格识别结果，希望对你有用
